std/random.d

```diff
@@ -1336,6 +1330,297 @@ version (AnyARC4Random)
     extern(C) private @nogc nothrow
     {
         uint arc4random() @safe;
+        void arc4random_buf(scope void* ptr, size_t len) @system;
+    }
+
+    /+
+    Tries to use a system API to get entropy. Resources are not cached
+    due to the expectation that this function will not be called more
+    than once per program execution.
+
+    Params:
+        ptr = address at which to start writing
+        len = number of bytes to write
+
+    Returns:
+    always `len` (does not fail)
```

Comment:
Nit: We typically indent for one-level after sections

Reply:
fixed